reconcile/openshift_rolebindings.py

```diff
@@ -110,7 +110,7 @@ def fetch_desired_state(ri, oc_map):
 
         for permission in permissions:
             cluster = permission['cluster']
-            if not oc_map.get(cluster):
+            if oc_map and not oc_map.get(cluster):
```

Comment:
is this correct? if `oc_map` is none we will not enter the `if`.

Perhaps you meant:
```
if not oc_map or (oc_map and not oc_map.get(cluster)):
```

Reply:
I mean to not enter the if when oc_map is none